Makefile

```diff
@@ -34,8 +34,8 @@ idb/postgres/internal/schema/setup_postgres_sql.go:	idb/postgres/internal/schema
 	cd idb/postgres/internal/schema && go generate
 
 idb/mocks/IndexerDb.go:	idb/idb.go
-	go install github.com/vektra/mockery/.../
-	cd idb && mockery -name=IndexerDb
+	go install github.com/vektra/mockery/v2@latest
+	cd idb && mockery --name=IndexerDb
```

Comment:
Why was this needed?

Reply:
I wasn't able to run the integration tests with the latest version of go.  I had to update the install command to include the `v2@latest` to get it.  The `--name` was done because `-name` was an invalid argument to the latest `mockery` program.